packages/flutter_tools/templates/plugin/windows.tmpl/pluginClassSnakeCase.cpp.tmpl

```diff
@@ -10,15 +10,14 @@
 #include <flutter/plugin_registrar_windows.h>
 #include <flutter/standard_method_codec.h>
 
-#include <memory>
```

Comment:
> Possibly unused #include derective

Reply:
You should file a bug against whatever tool gave you that warning, because it's used frequently in the file. It's used on the *very first line* of implementation code, in fact (line 22).